packages/main/src/components/ThemeProvider/ThemingSupport.ts

```diff
@@ -0,0 +1,12 @@
+import { root as sap_belize } from '@sap-theming/theming-base-content/content/Base/baseLib/sap_belize/variables.json';
+import { root as sap_belize_hcb } from '@sap-theming/theming-base-content/content/Base/baseLib/sap_belize_hcb/variables.json';
+import { root as sap_belize_hcw } from '@sap-theming/theming-base-content/content/Base/baseLib/sap_belize_hcw/variables.json';
+import { root as sap_fiori_3 } from '@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/variables.json';
```

Comment:
the fiori_3 theme can be removed from this map, it is included in the ThemeProvider already.
Looks like a merge issue?